### PR TITLE
[MOBILE-1590] Release 2.0.0

### PR DESCRIPTION
**Pull request by rlepinski**
Restructures the repo to be a monorepo so we can actually develop the module without copying files. Updates the Android module to depend on 6.0.0 version fo the gimbal adapter and fixes any compile issues. Tested with the example app.

### CODE REVIEW

File: src/android/build.gradle [hunk 1]
```diff
@@ -11,6 +11,6 @@ android {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation 'com.urbanairship.android:gimbal-adapter:5.0.0'
+    implementation 'com.urbanairship.android:gimbal-adapter:6.0.0'
```
Comment (rlepinski) [inline src/android/build.gradle:14]: main change

File: src/android/src/main/java/com/urbanairship/reactnative/gimbal/AirshipGimbalAdapterModule.java [hunk 2]
```diff
@@ -2,13 +2,15 @@
...
+import androidx.annotation.NonNull;
+
 import com.facebook.react.bridge.Promise;
 import com.facebook.react.bridge.ReactApplicationContext;
 import com.facebook.react.bridge.ReactContextBaseJavaModule;
 import com.facebook.react.bridge.ReactMethod;
 import com.gimbal.android.Visit;
+import com.urbanairship.analytics.location.RegionEvent;
 import com.urbanairship.gimbal.GimbalAdapter;
-import com.urbanairship.location.RegionEvent;
```
Comment (rlepinski) [inline src/android/src/main/java/com/urbanairship/reactnative/gimbal/AirshipGimbalAdapterModule.java:11]: main change

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# https://help.github.com/en/articles/about-code-owners
+*       @BrianBatchelder @marc-scig @crow @rlepinski
```
Comment (crow) [inline .github/CODEOWNERS]: paris ppl

**Review by crow — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
-# Urban Airship React Native
+# c
```
Comment (BrianBatchelder) [inline README.md]: ??
```suggestion
# Airship React Native
```

File: README.md [hunk 5]
```diff
@@ -66,6 +66,8 @@ notificationAccentColor = #ff0000
 
 ## Starting the adapter
 
+In order to start the adapter, you will need call `start` with the app's Gimbal API Key:
```
Comment (BrianBatchelder) [inline README.md]:
```suggestion
In order to start the adapter, your app will need to call `start` with the app's Gimbal API Key:
```

**Review by BrianBatchelder — APPROVED** [review]
Two minor things. Pre-approved.

File: README.md (hunk 4 above)
Comment (rlepinski) [reply README.md]: my command key is failing on my laptop. I probably tried to copy something

**Pull request merged**